File: src/lib/models/DTO/Domain/MeetingUserMappingDto.cs
using System;
using System.Collections.Generic;

namespace DTO.Domain
{
    public class MeetingUserMappingDto: BaseDomainDto
    {
        public long MeetingId{ get; set; }
        public long UserId{ get; set; }

        public UserDto User{ get; set; }
        public ICollection<MeetingReviewDto> Reviews{ get; set; }

        public MeetingUserMappingDto(): base(){ }
        public MeetingUserMappingDto(long meetingId, long userId): base()
        { 
            this.MeetingId = meetingId;
            this.UserId = userId;
        }
    }
}

File: src/ui/asq-ui/src/app/_models/domain/meeting-user-mapping-dto.ts
import { BaseDomainDto } from './base-domain-dto';
import { UserDto } from './user-dto';
import { MeetingReviewDto } from './meeting-review-dto';

export class MeetingUserMappingDto extends BaseDomainDto{
    public meetingId: number;
    public userId: number;
    public inactive: boolean;
    public user: UserDto;

    public reviews: MeetingReviewDto[];
}

